assets/src/util/calendar-utils.js

```diff
@@ -226,3 +228,23 @@ export function adjustAsyncResourcesBusinessHours(resources, calendarRef, date)
 
   return false;
 }
+
+/**
+ * @param {object} internalAsyncEvents Object of all resource events gathered async
+ * @returns {object} Object of all resource events gatered async, cleaned for duplicates
+ */
+export function removeDuplicateEvents(internalAsyncEvents) {
```

Comment:
This function seems very hackish... What is the purpose? Why are there duplicates? Instead of fixing an issue by applying cleanup, I think you have to tackle the issue at the point where you get the duplicated events.

Please add more comments.